stdlib/public/core/LifetimeManager.swift

```diff
@@ -73,18 +77,20 @@ public func _fixLifetime<T>(_ x: T) {
 ///     The pointer argument is valid only for the duration of the function's
 ///     execution.
 /// - Returns: The return value, if any, of the `body` closure.
-@inlinable
```

Comment:
I agree that `@_alwaysEmitIntoClient` implies `@inlinable`, but it seems the compiler has a hard time actually wanting to inline these things into callers. E.g. https://github.com/apple/swift/pull/71443 this added `@_transparent` which is different, but even in release a call to this didn't want to get inlined.

Reply:
`@_aeic` and `@inlinable` both allow the compiler to inline the function body into callers, but they should not at all differ in how much they encourage such inlining. If they do differ, then I think that'd be a bug.